ci-docker-base/Dockerfile.cilint-clang-tidy

```diff
@@ -0,0 +1,12 @@
+# ubuntu18.04-cuda10.2-py3.6-tidy11
+FROM nvidia/cuda:10.2-devel-ubuntu18.04
+
+RUN apt-get update && apt-get upgrade -y
+RUN apt install -y software-properties-common wget
+RUN wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add -
+RUN apt-add-repository "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-11 main"
+RUN apt-add-repository ppa:git-core/ppa
+
+RUN apt-get update && apt-get upgrade -y && apt-get install -y git python3-dev python3-pip build-essential cmake clang-tidy-11 time
+RUN update-alternatives --install /usr/bin/clang-tidy clang-tidy /usr/bin/clang-tidy-11 1000
+RUN pip3 install pyyaml typing_extensions dataclasses
```

Comment:
nit but could you add a trailing newline here? :P

Reply:
hmm. why didn't it failed quick check?

Reply:
because this isn't in pytorch/pytorch